common/on_action/wc_mana_on_actions.txt

```diff
@@ -0,0 +1,6 @@
+﻿wc_on_mana_regeneration = {
```

Comment:
I didn't notice mana regeneration happening.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/120373066-f915dd80-c328-11eb-905e-39c5b5885805.png)
![image](https://user-images.githubusercontent.com/46384992/120373088-fdda9180-c328-11eb-81a3-74206685dae3.png)

</details>

Reply:
Should be fixed now